addons/bus/static/tests/simple_notification_tests.js

```diff
@@ -44,7 +44,7 @@ QUnit.test("receive and display simple notification with specific type", async (
         message: "simple notification",
         type: "info",
     });
-    await contains(".o_notification.border-info");
+    await contains(".o_notification:has(.o_notification_bar.bg-info)");
```

Comment:
:warning: `:has` is still not supported by a stable version of Firefox (120 as of this review).
It would then make those tests break in that browser.

**Update**: you're very lucky: Firefox 121 (which supports `:has`) has just been released today! :nerd_face: